src/plugins/telemetry/server/fetcher.ts

```diff
@@ -213,7 +212,6 @@ export class FetcherTask {
   private async fetchTelemetry() {
     return await this.telemetryCollectionManager!.getStats({
       unencrypted: false,
-      timestamp: moment().valueOf(),
```

Comment:
I don't know how it slipped in the previous PR (#83791) 😱 

TS builds should have complained at least! 🤔